src/averageAges.js

```diff
@@ -55,7 +77,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const onlyChildren = people.filter(({ mother }) => (
```

Comment:
And same here.
Use `onlyWithSon` argument for conditional filtering and make your code shorter